terraform/projects/app-transition-postgresql/main.tf

```
@@ -137,6 +138,8 @@ module "transition-postgresql-standby_rds_instance" {
   instance_name              = "${var.stackname}-transition-postgresql-standby"
   security_group_ids         = ["${data.terraform_remote_state.infra_security_groups.sg_transition-postgresql-standby_id}"]
   create_replicate_source_db = "1"
+  allocated_storage          = "120"
+  max_allocated_storage      = "150"
```

Leaving aside the question of the default, I do think we should make the max several times bigger than the current, otherwise we only get the downsides of autoscaling without the benefit.

```suggestion
  max_allocated_storage      = "500"
```